src/open_inwoner/scss/components/Contacts/Contacts.scss

```diff
@@ -0,0 +1,57 @@
+.approval {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--color-accent-lighter);
+  margin-top: var(--gutter-width);
+  border-radius: var(--border-radius);
+
+  &__header {
+    padding: var(--spacing-medium);
+  }
+
+  &__single {
+    display: flex;
+    flex-direction: column;
+    padding: var(--spacing-medium);
+    margin: 0;
+  }
+
+  &__list {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    padding: var(--spacing-medium);
+    margin: 0;
+  }
+
+  &__list-item {
+    display: flex;
+    align-items: center;
+    border-bottom: var(--border-width) solid var(--color-mute);
+
+    p,
+    form {
+      flex: 1;
+    }
+  }
+
+  p {
+    margin: 0;
+  }
+
+  .btn-reject {
```

Comment:
Since it's a button class, I followed the existing pattern in the project (button--danger, button--disabled etc.) and removed this rule from contacts scss file. I added it to the button.scss.